src/commands/ManagedEnvironmentContext.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { type ManagedEnvironment } from "@azure/arm-appcontainers";
+import { type ISubscriptionActionContext } from "@microsoft/vscode-azext-utils";
+import { type AzureSubscription } from "@microsoft/vscode-azureresources-api";
+
+export interface RequiredManagedEnvironmentContext extends ISubscriptionActionContext {
```

Comment:
I used the name "Required" here to help quickly identify that the managed environment resource is not optional.

Reply:
Discussed offline, but I think we settled on using a naming convention that would put the resource ala `ManagedEnvironment` first.